ci-build/test.sh

```diff
@@ -8,12 +8,26 @@ panic() {
     exit 1
```

Comment:
I don't think this file is used in CI check. Now CI steps are written in job configurations.

Reply:
Where is the CI code?

Reply:
Maybe you can ask @zhouqiang-cl for help.